src/components/seo.jsx

```diff
@@ -83,6 +83,13 @@ export function Seo({
         sizes="180x180"
         href="/apple-touch-icon.png"
       />
+      {/* This line can be removed */}
```

Comment:
Do we need the "This line can be removed" comment?

Reply:
I was worried about this addition becoming dead code for anyone who forks the starter, but this wording might not be great... What if it said "For demonstration only" instead?

Reply:
Also worth noting that this meta tag would NOT be good for SEO on an actual production site

Reply:
Hmm what if it said something like "The meta robots tag below is for demonstration only"? Something that makes it explicit what line the comment is referring to.

Also I might have missed this, but what was the value add of this change (if it's bad for SEO)?

Reply:
```suggestion
      {/* The following meta tag is for demonstration only and can be removed */}
```
Thanks for the suggestion!

This change is to prevent the demo site, which contains a lot of generated/fake date, from being indexed by Google/etc.